app/assets/stylesheets/blacklight.scss

```diff
@@ -66,3 +66,38 @@
     }
   }
 }
+
+dt.blacklight-full_text_tesimv {
+  @extend .h4;
+  display: inline-block;
+  float: none;
+  text-align: initial;
+  width: auto;
+
+  // collapse carets shamelessly stolen from blacklight facets
+  &[data-toggle="collapse"]:after {
+    color: #808080;
```

Comment:
Color literals like `#808080` should only be used in variable declarations; they should be referred to via variable everywhere else.